.github/workflows/release.yml

```diff
@@ -30,7 +30,9 @@ jobs:
 
       - name: auto-dist-tag
         run: npx auto-dist-tag@1 --write
+        working-directory: qunit-dom
```

Comment:
Here the working-directory makes sense to me, so I'd not change that.

Note; Although on another note we should consider setting up scripts for that in the root package.json.

Reply:
done